src/codegen/sys/build.rs

```diff
@@ -22,10 +23,13 @@ pub fn generate(env: &Env) {
 }
 
 fn generate_build_script(w: &mut Write, env: &Env) -> Result<()> {
+    try!(general::start_comments(w, &env.config));
     try!(write!(
         w,
         "{}",
-        r##"extern crate pkg_config;
+        r##"
+
```

Comment:
Please make only one empty line with `try!(writeln!(w));` instead

Reply:
That's weird. Why? 

(also why use try! when the crate requires Rust 1.19?)

Reply:
Updated PR